hs_core/management/commands/solr_build_json.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+
+"""
+This calls all preparation routines involved in creating SOLR records.
```

Comment:
From Alva: This is obsolete. I was trying to speed up SOLR response. This approach failed.